### PR TITLE
Main styles

### PR DESCRIPTION
**Pull request by galinafrost**

### CODE REVIEW

File: src/styles/scss/_config.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+$main-font: "Roboto", sans-serif;
+$tipical-font-size: 12px;
+
+$white-cl: rgba(255, 255, 255, 1);
+$blue-cl: rgba(246, 247, 251, 1);
+$main-txt-cl: rgba(82, 85, 95, 1);
+$second-txt-cl: rgba(0, 0, 0, 1);
+$btn-active-cl: rgba(255, 117, 29, 1);
+$btn-noactive-cl: rgba(245, 246, 251, 1);
```
Comment (lelchek) [inline src/styles/scss/_config.scss:9]: btn-inactive-cl

File: src/styles/scss/_mixins.scss [hunk 2]
```diff
@@ -0,0 +1,22 @@
+@mixin mobile {
+  @media screen and (min-width: 320px) {
```
Comment (lelchek) [inline src/styles/scss/_mixins.scss]: It would be great to set the values ​​of the min-width (320, 772, 1200) with variables

**Review by lelchek — CHANGES_REQUESTED** [review]

File: src/styles/scss/_config.scss (hunk 1 above)
Comment (galinafrost) [reply src/styles/scss/_config.scss:9]: done

File: src/styles/scss/_mixins.scss (hunk 2 above)
Comment (galinafrost) [reply src/styles/scss/_mixins.scss]: done

**Pull request merged**